lib/service/service.go

```diff
@@ -1557,10 +1557,10 @@ func (process *TeleportProcess) newLocalCacheForRemoteProxy(clt auth.ClientI, ca
 	return process.newLocalCache(clt, cache.ForRemoteProxy, cacheName)
 }
 
-// DELETE IN: 5.1.
+// DELETE IN: 8.0.0
 //
 // newLocalCacheForOldRemoteProxy returns new instance of access point
-// configured for an old remote proxy.
+// configured for a pre-v7 remote proxy.
```

Comment:
Nit: TBH I think the previous "is old" / "an old" terminology was fine since now we have to update all these comments and function names with each release whereas before we would just update version in the "isOld" function and that's it.

Reply:
My general conviction is that how code is written should favour readability as opposed to "editability" (when the two are in conflict) since code is attempted to be understood more often than it gets edited.

I can easily revert this though.

Reply:
Yeah that's fair and I agree from readability standpoint it's better. Let's keep your version.